modules/integrate-with-tenable.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-image-vulnerability-scanners.adoc
+:_module-type: PROCEDURE
+[id="integrate-with-tenable_{context}"]
+= Integrating with Tenable
+
+You can integrate {product-title} with Tenable for scanning images.
+
+.Prerequisites
+* You need an *Access Key* and a *Secret Key* for authentication.
```

Comment:
You could update this prerequisite to clarify which platform this relates to (i.e. RHACS or Tenable).